### PR TITLE
Fixed a missing package in Maven and Gradle code analysis.

### PR DESCRIPTION
**Pull request by HSAR**

### CODE REVIEW

File: Tasks/Gradle/package.json [hunk 1]
```diff
@@ -8,8 +8,9 @@
     "url": "https://github.com/Microsoft/vso-agent-tasks/issues"
   },
   "dependencies": {
-    "xml2js": "^0.4.16",
+    "glob": "^7.1.0",
```
Comment (bgavrilMS) [inline Tasks/Gradle/package.json]: Remove the ^ - the carret means that on build node will install any 7.x version of glob. But we only have LCA approval for 7.1.0

Comment (bgavrilMS) [inline Tasks/Gradle/package.json]: Should we bump the task version just to be safe?

Comment (HSAR) [inline Tasks/Gradle/package.json]: Fixed.

Comment (HSAR) [inline Tasks/Gradle/package.json]: Done.

**Pull request merged**